app/views/dashboard_admin/dashboard_admin.html.erb

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dashboard admin</title>
+</head>
+<body>
+  <nav class="navbar">
+      <div class="container-fluid">
+        <span class="navbar-brand mb-0 h1" id="logo">
+            <%= image_tag "foocorp.svg" =%>
+        </span>
+
+        <div class="user_admin">
+          <%= image_tag "admin.svg" =%>
+          <a href="">Admin</a>
+        </div>
+      </div>
+    </nav>
+
+    <div class="main">
+      <div class="management-container">
+        <h1>Management</h1>
+        <input type="button" value="Employees">
+        <input type="button" value="Company branches">
+      </div>
+      <div class="reports-container">
+        <h1>Reports</h1>
+        <input type="button" value="Attendance by day">
+        <input type="button" value="Average time checked in/out by month">  
+        <input type="button" value="Absence report by month">
+      </div>
+    </div>
+</body>
+</html>
```

Comment:
Why did you leave this code here? Remember that this part is already in the application.erb

Reply:
Done.